From 1c42bb5e34783ea7170e96905d9d60e07e23933f Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Mon, 27 Jan 2014 21:58:44 -0500
Subject: [PATCH] Avoid a compiler warning

gcc doesn't like returning without a value from a non-void
function.
---
 gtk/gtkselection.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtkselection.c b/gtk/gtkselection.c
index 1c77002..3fa8971 100644
--- a/gtk/gtkselection.c
+++ b/gtk/gtkselection.c
@@ -2294,7 +2294,7 @@ _gtk_selection_request (GtkWidget *widget,
   gulong selection_max_size;
 
   if (event->requestor == NULL)
-    return;
+    return FALSE;
 
   if (initialize)
     gtk_selection_init ();
-- 
1.8.3.rc0

